### PR TITLE
ceph-disk: prevent unnecessary tracebacks from subprocess.check_call

### PR DESCRIPTION
**Pull request by alfredodeza**
See: http://tracker.ceph.com/issues/16125

### CODE REVIEW

**Comment by dachary** [comment]
jenkins test this please (no logs)

**Comment by dachary** [comment]
<pre>
value = AssertionError("Expected call: command_check_call(['sgdisk', '--largest-new=1'...3-4f30-b4c6-b4b80ceff106', '--mbrtogpt', '--', '/dev/wholedisk'], exit=True)",)
from_value = None

    def raise_from(value, from_value):
>       raise value
E       AssertionError: Expected call: command_check_call(['sgdisk', '--largest-new=1', '--change-name=1:ceph journal', '--partition-guid=1:UUID', '--typecode=1:45b0969e-9b03-4f30-b4c6-b4b80ceff106', '--mbrtogpt', '--', '/dev/wholedisk'])
E       Actual call: command_check_call(['sgdisk', '--largest-new=1', '--change-name=1:ceph journal', '--partition-guid=1:UUID', '--typecode=1:45b0969e-9b03-4f30-b4c6-b4b80ceff106', '--mbrtogpt', '--', '/dev/wholedisk'], exit=True)

../src/ceph-disk/.tox/py27/local/lib/python2.7/site-packages/six.py:718: AssertionError
</pre>

**Comment by alfredodeza** [comment]
@dachary I updated the tests to pass

File: src/ceph-disk/ceph_disk/main.py [hunk 1]
```diff
@@ -1,12 +1,10 @@
 #!/usr/bin/env python
 #
-# Copyright (C) 2015, 2016 Red Hat <contact@redhat.com>
+# Copyright (C) 2015-2016 Red Hat <contact@redhat.com> Authors
 # Copyright (C) 2014 Inktank <info@inktank.com>
 # Copyright (C) 2014 Cloudwatt <libre.licensing@cloudwatt.com>
 # Copyright (C) 2014 Catalyst.net Ltd
 #
-# Author: Loic Dachary <loic@dachary.org>
-#
```
Comment (smithfarm) [inline src/ceph-disk/ceph_disk/main.py]: Loic is the primary author of ceph-disk, so please don't drop these two lines.

File: src/ceph-disk/ceph_disk/main.py [hunk 2]
```diff
@@ -17,6 +15,7 @@
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU Library Public License for more details.
 #
+# See the AUTHORS file for names of contributors
```
Comment (smithfarm) [inline src/ceph-disk/ceph_disk/main.py]: ...for the names of additional contributors

**Comment by smithfarm** [comment]
@alfredodeza You could also just drop the commit, since it's entirely unrelated to the subject of the PR. The appropriate context for such a change, IMO, would be a separate PR implementing a global policy throughout the source tree. Since there apparently is no such global policy, that would need to be determined first, preferably by discussing in the mailing list and reaching a consensus among all Ceph developers.

File: src/ceph-disk/ceph_disk/main.py [hunk 3]
```diff
@@ -1,12 +1,10 @@
 #!/usr/bin/env python
 #
-# Copyright (C) 2015, 2016 Red Hat <contact@redhat.com>
+# Copyright (C) 2015-2016 Red Hat <contact@redhat.com> Authors
```
Comment (smithfarm) [inline src/ceph-disk/ceph_disk/main.py]: Does Red Hat recommend that the copyright statement look like this? (The addition of the word "Authors" seems to render the copyright statement ambiguous. To whom does the copyright belong - Red Hat, or its Authors?)

File: src/ceph-disk/ceph_disk/main.py (hunk 1 above)
Comment (dachary) [inline src/ceph-disk/ceph_disk/main.py]: Please keep this line (the rationale is explained at http://tracker.ceph.com/issues/16125#note-10)

**Comment by alfredodeza** [comment]
Closing this, as it is being required to have separate pull requests.

**Pull request closed without merging**